.claude/skills/auction-sim/references/auction-rules.md

```
@@ -0,0 +1,158 @@
+  ├ 구매자가 입금 완료 알림 (paymentConfirmed=true)
+  └ 판매자가 확인 (paymentVerified=true) 또는 거절 (paymentConfirmed=false로 되돌림)
+SHIPPED
+  └ 판매자가 송장 입력 (courierCompany, trackingNumber)
+DELIVERED
+  └ 구매자가 수령 확인 → Trade status → ARRANGED
+```
+
+송장 입력 기한: `paymentVerified` 후 **72시간** (`Trade.SHIPPING_DEADLINE_HOURS`).
+
+### 10.6 직거래 플로우 (DirectTradeStatus)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)